Source/Core/InputCommon/ControllerEmu/ControlGroup/Cursor.cpp

```diff
@@ -33,7 +33,7 @@ Cursor::Cursor(std::string name, std::string ui_name)
 
   AddInput(Translate, _trans("Relative Input Hold"));
 
-  // Default values are optimized for "Super Mario Galaxy 2".
+  // Default values are optimized for the Wii System Menu.
```

Comment:
Thanks for changing the comment but it isn't really accurate. :P
Defaults of 20/15 might allow reaching every area of the screen but it certainly isn't "optimized" for the system menu.
There is extreme overshoot.